.github/workflows/ci.yml

```
@@ -6,6 +6,7 @@ on:
     - master
 
   pull_request:
+    paths: ["*.rs", "*.toml", "*.json", "*.lua", "*.luau", "*.rbxm", "*.rbxl", "*.rbxmx", "*.rbxlx", "*.txt", "*.yml", "*.csv"]
```

CI should run when test files have changed:
```suggestion
    paths: ["*.rs", "*.toml", "*.json", "*.lua", "*.luau", "*.rbxm", "*.rbxl", "*.rbxmx", "*.rbxlx", "*.txt", "*.yml", "*.csv", "*.snap"]
```